src/components/DuffelNGSView/lib/index.ts

```diff
@@ -26,23 +26,23 @@ export const NGS_SHELF_INFO: Record<OfferSlice["ngs_shelf"], ShelfInfo> = {
   },
   3: {
     short_title: "Plus",
-    full_title: "Economy Plus",
+    full_title: "Plus",
     description:
-      "Economy seats with extra leg room that allow seat selection, flexible change and cancellation options and checked baggage.",
+      "Preferred seating that offer more comfort to passengers, such as additional legroom, width or having the middle seat free for the row.",
```

Comment:
```suggestion
      "Preferred seating that offers more comfort to passengers, such as additional legroom, width or having the middle seat free for the row.",
```

This sounds more correct to me but feel free to ignore...

Reply:
Oh you're right 👍